JRuby.md

```diff
@@ -0,0 +1,117 @@
+## JRuby
+
+> The Ruby Programming Language on the JVM
+
+This document serves as a getting-started guide for those wishing to use
+[Sorbet](https://sorbet.org) with [JRuby](https://www.jruby.org/).
+
+The use of Sorbet on the JVM is not **officially** supported by the Sorbet
+development team however this document services as a getting started guide
+for those who wish to trot on a unsupported path.
```

Comment:
Can you add a note here that being not supported means bug reports that only reproduce on JRuby are likely to not get fixed or potentially even closed.

Reply:
Will add